src/shared/components/chevron/Chevron.scss

```diff
@@ -3,6 +3,7 @@
 .wrapper {
   display: inline-block;
   font-size: 0; // to prevent line-height's impact on the height of the element
+  cursor: pointer;
```

Comment:
pointer means an invitation to click. The clickable chevron has now been moved to ChevronButton, I hope. This chevron, in contrast, gets wrapped in a span (which it shouldn't be, but that's for another PR); which means that it isn't clickable, and it shouldn't change the cursor to pointer.